Tasks/DownloadBuildArtifactsV0/download_helper.ts

```diff
@@ -0,0 +1,98 @@
+import { debug as writeDebugInfo, loc as mapLocString } from 'azure-pipelines-task-lib/task';
```

Comment:
nit: Let's use original method names for consistency. It would be easier to switch between tasks in this case.

Reply:
Fixed it.